airflow/www/app.py

```diff
@@ -66,8 +66,16 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     flask_app = Flask(__name__)
     flask_app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
-    session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
-    flask_app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=session_lifetime_days)
+    if conf.has_option('webserver', 'SESSION_LIFETIME_DAYS'):
+        warnings.warn('SESSION_LIFETIME_DAYS option is deprecated. Please use `SESSION_LIFETIME_MINUTES`',
+                      DeprecationWarning)
+
+    if conf.has_option('webserver', 'FORCE_LOG_OUT_AFTER'):
+        warnings.warn('FORCE_LOG_OUT_AFTER option is deprecated. Please use `SESSION_LIFETIME_MINUTES`',
```

Comment:
not deprecated, removed. can remove this and just populate the UPDATING.MD

Reply:
I believe that we should warn user about removed options. Warning can be removed in next Airflow release.  
I can changed message to `option is removed. Please use ...`
Please let me know if I should remove this warning.

Reply:
option is removed. Please use ...